test/storage/file.js

```diff
@@ -1025,6 +1025,22 @@ describe('File', function() {
     });
   });
 
+  describe('makePublic', function() {
+    it('should make the file public', function(done) {
+
+      // If this mocked method (THAT DOES NOTHING!) is removed, nothing works.
+      file.acl.makeAclObject_ = function() {};
```

Comment:
That makes sense. Just doesn't make sense that I added an empty function and it still worked.